packages/x/src/studio/models/Field.ts

```diff
@@ -119,6 +119,12 @@ export class Field extends BaseModel<Field> {
    */
   collection: Collection;
 
+  /**
+   * The sharedmodel it belongs to. .
+   * @cleanable
+   */
+  sharedCollection?: SharedModel;
```

Comment:
The name here is kinda confusing since we have "Collections" behave different from models, let's do a getSharedModel() which does this check for us and retrieves us the SharedModel, we can always get it dynamically.